### ISSUE TITLE
1.12.2

### ISSUE THREAD

**Issue by Bluy6444** [DrZhark/mocreatures#169]
how come Mo' Creatures is updated to 1.12.2, but the relation mod Custom spawners still for 1.10!
please update? this mod has some really cool mobs

**Comment by ExDomino** [comment]
The link of the newest version of Custom Mob Spawners is on the Mo' Creatures Website:
https://www.mocreatures.org/downloads

Beware: it is protected behind three layers of adfly links.
You have to download a good adfly skiper in order to download it.

**Comment by CipheredEntity** [comment]
The fact that Custom Spawner isn't updated on Curse means that nobody can really add this mod to a mod pack for 1.12 on Curse though, which is really unfortunate.

**Comment by iStar5** [comment]
Yeah I was wanting to add it to my pack as well... I added it in manually but if someone wants to get your modpack then they will have to download it manually through the adfly layers as well unless I uploaded it onto my own OneDrive or something but even that is still too much trouble for most people. So I will be getting a different creatures pack instead sadly.
